plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -210,6 +210,16 @@ SecRule REQUEST_FILENAME "@endsWith /index.php" \
             "t:none,\
             ctl:ruleRemoveTargetById=942100;ARGS"
 
+# Cannot update page|post in WordPress due to `x-http-method-override` header
+SecRule REQUEST_FILENAME "@rx /wp-json/wp/v[0-9]+/(?:posts|pages|users)" \
```

Comment:
Does this situation only occur with POST / PUT requests? If so, it would be nice to restrict this rule to those methods only (by using a chain).